content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
```

Comment:
~Shows~ How to manage webhooks in Istio through istioctl.

Reply:
Revised accordingly.